tool_commands/create_breach_locations/breach_location_dialog.py

```diff
@@ -0,0 +1,79 @@
+from pathlib import Path
+from PyQt5.QtWidgets import QDialog
+from qgis.core import QgsProject
+from qgis.PyQt import uic
+from ThreeDiToolbox.tool_commands.create_breach_locations import breach_location
+
+import logging
+
+
+logger = logging.getLogger(__name__)
+
+ui_file = Path(__file__).parent / "breach_location_dialog.ui"
+assert ui_file.is_file()
+FORM_CLASS, _ = uic.loadUiType(ui_file)
+
+
+class CreateBreachLocationsDialogWidget(QDialog, FORM_CLASS):
+    def __init__(self, parent=None, command=None):
+        """Constructor
+        Args:
+            parent: Qt parent Widget
+            iface: QGiS interface
+            ts_datasource: TimeseriesDatasourceModel instance
```

Comment:
Op dit punt zal  je straks misschien een migratieconflict krijgen. (ts_datasourceS).
(Of ik, afhankelijk van hoe snel je merget :-) )